x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_utilities.tsx

```diff
@@ -122,7 +122,7 @@ export const formatter = new Intl.DateTimeFormat(i18n.getLocale(), {
   second: '2-digit',
 });
 
-const invalidDateText = i18n.translate(
+export const invalidDateText = i18n.translate(
```

Comment:
:grey_question: doc comment needed

Reply:
I don't think we should export string literals like this. Let's just move it to where it is being used.

Reply:
No exporting anymore, but will add